integration-tests/kubernetes/maven-invoker-way/src/it/minikube-with-existing-manifest/src/main/kubernetes/minikube.json

```diff
@@ -5,7 +5,7 @@
     "metadata": {
       "labels": {
         "app.kubernetes.io/name": "postgres",
-        "app.kubernetes.io/version" : "10.6"
+        "app.kubernetes.io/version" : "14.1"
```

Comment:
My guess is that this file shouldn't be changed and should be considered a static resource.

Reply:
PR in https://github.com/quarkusio/quarkus/pull/22165